swig/include/Relationship.i

```diff
@@ -19,22 +19,6 @@
 #ifndef SWIGCSHARP
 typedef int GDALRelationshipCardinality;
 typedef int GDALRelationshipType;
-#else
-%rename (RelationshipCardinality) GDALRelationshipCardinality;
```

Comment:
if you remove those type definitions I suspect the methods that use them will not be usable

Reply:
Those were already defined here https://github.com/szekerest/gdal/blob/515757f255066f76ea2bc42c255a777cdca01065/swig/include/gdal.i#L250
causing duplicate definition warnings.